.circleci/images/primary/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# using legacy image for phantomjs
```

Comment:
This file is purely for visibility, following the practice recommended by CircleCi. I've set up a Confluence doc on how to update the image: 
https://cm.usa.gov/confluence/display/SRCH/How+to+update+the+custom+Docker+image+used+by+CircleCi